x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_color_property.test.tsx

```diff
@@ -642,7 +642,7 @@ test('Should read out ordinal type correctly', async () => {
 });
 
 describe('renderDataMappingPopover', () => {
-  test('Should enable toggle when field is backed by geojson-source', () => {
+  test('Switch toggle should be enabled', () => {
```

Comment:
Since switch can never be disabled, can your rename test to better describe the test case. "Should render OrdinalDataMappingPopover"